packages/esm-service-queues-app/src/patient-info/patient-info.component.tsx

```diff
@@ -12,6 +12,7 @@ import {
   PatientBannerToggleContactDetailsButton,
   PatientBannerContactDetails,
 } from '@openmrs/esm-framework';
+import { displayName } from '@openmrs/esm-utils';
```

Comment:
now that #1010 is fixed, this import can be simplified; use @openmrs/esm-framework